english/explore-outside/investigate-marshmallow-structure/greet-green/greet-green.md

```diff
@@ -1,5 +1,8 @@
 You find yourself a way to the tower to greet the friendly looking green people. But their
-reaction is not quite what you expected. Upon seeing you, they cease their dance and douse 
```

Comment:
This modification looks good to me, so I'll go ahead and merge this. :baby_chick: